tests/test_release.py

```diff
@@ -168,7 +169,6 @@ def test_sdist_contents(build: Build, version: str):
             "pyproject.toml",
             "pytest_httpserver",
             "README.md",
-            "setup.py",
```

Comment:
No longer included in the wheel built with Poetry 1.4.2

Reply:
Ok, thanks.
This is also to know that setup.py is no longer included in the wheel, and we are now accepting it. But the test ensures that we are aware of this.